assets/src/components/detours/diversionPanel.tsx

```diff
@@ -66,6 +66,15 @@ export const DiversionPanel = ({
                   {d.instruction}
                 </ListGroup.Item>
               ))}
+              {detourFinished && (
+                <ListGroup.Item
+                  key="return-to-route"
+                  as="li"
+                  className="c-diversion-panel__conclude-detour-text"
+                >
+                  Return to Regular Route
+                </ListGroup.Item>
```

Comment:
suggestion: would using `<strong/>` work instead?
```suggestion
                >
                  <strong>Return to Regular Route</strong>
                </ListGroup.Item>
```

Reply:
Not quite, because `<strong/>` maps to `font-weight: bold`, while the [Figma](https://www.figma.com/file/XGmmwVROXdOvnWtJeUFC50/Skate-%7C-Unplanned-Detours-%7C-Hand-Off?type=design&node-id=252-3825&mode=design&t=N13hE4ik6aMXmEX9-0) calls for medium font-weight

Reply:
I personally think that's okay, but we should loop in design. My thinking is either the designs should be updated to accommodate `<strong>`'s default value, or we update the styling of `<strong>` to match what Figma says.

For now, so we can merge this, can we use `<strong>` but set a class on it to set the font weight to what it says in Figma? Otherwise I think this PR would be blocked by a round trip through design.

```suggestion
                >
                  <strong className="c-diversion-panel__conclude-detour-text">
                    Return to Regular Route
                  </strong>
                </ListGroup.Item>
```

Reply:
*Question*: What's the benefit of using `<strong />` if we're overriding its styling anyway? Does it provide an accessibility assist?

Reply:
ooh I found a better solution!
```suggestion
                >
                  <strong className="fw-medium">
                    Return to Regular Route
                  </strong>
                </ListGroup.Item>
```

Reply:
IMO `<strong>` is semantically correct for this context _and_ communicates that the text is emphasized via the accessibility tree

Reply:
Ay I like this! Makes sense to me. Thanks 👏